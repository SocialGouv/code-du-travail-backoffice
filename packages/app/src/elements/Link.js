import styled from "@emotion/styled";

export default styled.button`
  background: transparent;
  border: 0;
  color: var(--color-text-blue);
  cursor: pointer;
  padding: 0;
  text-decoration: underline;
`;
