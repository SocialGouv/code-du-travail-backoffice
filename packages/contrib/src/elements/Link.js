import styled from "styled-components";

export default styled.button`
  background: transparent;
  border: 0;
  color: var(--color-text-blue);
  cursor: pointer;
  padding: 0;
  text-decoration: underline;
`;
