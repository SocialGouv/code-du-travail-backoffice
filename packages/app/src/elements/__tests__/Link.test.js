import React from "react";
import { create } from "react-test-renderer";

import Link from "../Link";

describe("elements/<Link />", () => {
  it(`should pass`, () => {
    const $link = create(<Link />);

    expect($link).toHaveTestRenderedType("button");
  });
});
