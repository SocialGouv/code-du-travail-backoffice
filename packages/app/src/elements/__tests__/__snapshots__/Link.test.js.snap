// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/<Link /> should pass 1`] = `
.emotion-0 {
  background: transparent;
  border: 0;
  color: var(--color-text-blue);
  cursor: pointer;
  padding: 0;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<button
  className="emotion-0 emotion-1"
>
  A Link
</button>
`;
